content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.5.0.properties

```diff
@@ -1,6 +0,0 @@
-driverVersion=v2.5.0
```

Comment:
Any specific reason to delete these properties ?

Reply:
We are supporting the installation wizard for csm version 1.6 and onwards